### PR TITLE
IE11 compatible

### PR DESCRIPTION
**Pull request by nullpointer2017**
ie11 compatible

### CODE REVIEW

File: app/javascript/mastodon/actions/store.js [hunk 1]
```diff
@@ -1,11 +1,11 @@
 import Immutable from 'immutable';
 
 export const STORE_HYDRATE = 'STORE_HYDRATE';
-
+const NumberIsNaN = (number) => typeof number === 'number' && isNaN(number);
 const convertState = rawState =>
   Immutable.fromJS(rawState, (k, v) =>
     Immutable.Iterable.isIndexed(v) ? v.toList() : v.toMap().mapKeys(x =>
-      Number.isNaN(x * 1) ? x : x * 1));
+      NumberIsNaN(x * 1) ? x : x * 1));
```
Comment (ykzts) [inline app/javascript/mastodon/actions/store.js]: `babel-polyfill` is included [`Number.isNaN`](https://github.com/zloirock/core-js/blob/v2.4.1/modules/es6.number.is-nan.js).

File: app/javascript/packs/application.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import 'font-awesome/css/font-awesome.css';
 import '../styles/application.scss';
-
+require("babel-polyfill");
```
Comment (ykzts) [inline app/javascript/packs/application.js]: You should not `require` anything not included in package.json.

Comment (ykzts) [inline app/javascript/packs/application.js]: `babel-polyfill` also "`require`" unnecessary things. You should use "useBuiltIns: true" of `babel-preset-env`.

**Review by ykzts — CHANGES_REQUESTED** [review]

**Comment by ykzts** [comment]
In my opinion, I think that there is no merit to support the environment released three years ago like IE 11.

**Comment by Gargron** [comment]
I agree with @ykzts

**Pull request closed without merging**